WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/CardType.kt

```diff
@@ -0,0 +1,7 @@
+package org.wordpress.android.ui.mysite.cards.dashboard
+
+enum class CardType {
```

Comment:
1, 2, 3, 4:  Everything is accepted in 2a847fb

I really liked this suggestion 3., 
>  I would recommend putting the ERROR_CARD as the first value of this enum class  so then when a new value is added, which normally goes below the already existing values, it will not break the flow, with an error type in between normal types.

I placed it at the last to give lesser importance to the error card than the content cards but your suggestion is more valuable. Thank you!

Reply:
You are plainly the best, thank YOU! 🥇